mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/utbetaling/db/UtbetalingQueries.kt

```diff
@@ -53,7 +53,7 @@ class UtbetalingQueries(private val session: Session) {
             "periode_start" to dbo.periode.start,
             "periode_slutt" to dbo.periode.slutt,
             "beregningsmodell" to when (dbo.beregning) {
-                is UtbetalingBeregningAft -> Beregningsmodell.FORHANDSGODKJENT.name
+                is UtbetalingBeregningForhandsgodkjent -> Beregningsmodell.FORHANDSGODKJENT.name
                 is UtbetalingBeregningFri -> Beregningsmodell.FRI.name
             },
```

Comment:
Vi har også en `Prismodell`-enum som er lagret på avtaler for øyeblikket og som  foreløpig også er `FORHANDSGODKJENT | FRI`. Burde vi erstatte `beregningsmodell` med `prismodell`, eller er det to forskjellige ting?